Assets/Modules/0_Global/Scripts/Utils.cs

```diff
@@ -66,7 +67,7 @@ public static int RandomInt(int min, int max)
         public static float RandomFloat(float min, float max)
         {
             double val = (random.NextDouble() * (max - min) + min);
-            return (float) val;
+            return (float)val;
```

Comment:
```suggestion
            return (float) val;
```
Espace à mettre lors du cast